include/sys/ringbuf.h

```diff
@@ -23,6 +23,8 @@ static inline bool ringbuf_full(ringbuf_t *buf) {
 }
 
 void ringbuf_init(ringbuf_t *rb, void *buf, size_t size);
+void ringbuf_produce(ringbuf_t *buf, unsigned bytes);
+void ringbuf_consume(ringbuf_t *buf, unsigned bytes);
 bool ringbuf_putb(ringbuf_t *buf, uint8_t byte);
```

Comment:
Minor: you could change `unsigned` to `size_t` to stay consistent with the rest of `ringbuf_*` interface

Reply:
thanks, done